app/assets/javascripts/modules/users/components/table-row.vue

```diff
@@ -69,14 +69,17 @@
       },
 
       toggleEnabled() {
-        UsersService.toggleEnabled(this.user).then(() => {
+        UsersService.toggleEnabled(this.user).then((response) => {
           set(this.user, 'enabled', !this.user.enabled);
 
           if (this.user.enabled) {
             this.$alert.$show(`User '${this.user.username}' has been enabled`);
           } else {
             this.$alert.$show(`User '${this.user.username}' has been disabled`);
           }
+          if (response.data.redirect) {
+            window.location.reload(false);
+          }
```

Comment:
Did any test failed to make you add this? Because from the current UI state there's no way of the disabling the current user.

Reply:
My bad. It's possible with multiple admins. Nice catch!

Reply:
Nothing broke afaik. On the backend we log out the user, so the session will be marked as invalid, and upon refresh everything will be ok.